keps/sig-auth/3331-structured-config-for-oidc-authentication/kep.yaml

```diff
@@ -0,0 +1,20 @@
+name: 3331-structured-config-for-oidc-authentication
+title: Structured config for OIDC authentication
+kep-number: "3331"
+authors: ['@nabokihms']
+owning-sig: sig-auth
+participating-sigs: [sig-auth]
+reviewers: ['@enj', '@TBD']
+approvers: ['@TBD']
+prr-approvers: []
+creation-date: "2022-06-02"
+status: provisional
+stage: alpha
+latest-milestone: "v1.26"
+milestone:
+    alpha: "v1.26"
```

Comment:
Should alpha be "v1.27" since I saw a comment "/milestone v1.27"?

Reply:
Yeah, you are right.  I will change it.

Reply:
I redirected this KEP to milestone 1.27, thanks.